frontend/packages/console-shared/src/components/status/StatusIconAndText.tsx

```diff
@@ -8,15 +8,22 @@ type StatusIconAndTextProps = {
   title?: string;
   spin?: boolean;
   iconOnly?: boolean;
+  noTooltip?: boolean;
 };
 
-const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({ icon, title, spin, iconOnly }) => {
+const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({
+  icon,
+  title,
+  spin,
+  iconOnly,
+  noTooltip,
+}) => {
   if (!title) {
     return <>{DASH}</>;
   }
 
   return (
-    <span className="co-icon-and-text" title={title}>
+    <span className="co-icon-and-text" title={title && !noTooltip ? title : undefined}>
```

Comment:
IIUC it should be: 
```suggestion
    <span className="co-icon-and-text" title={iconOnly && !noTooltip ? title : undefined}>
```
title not being defined is handled by the if statement above, here we want to ensure that the title is added only when we're rendering only icon (`iconOnly` is true). So the information is not duplicated in the icon title and status title.

Reply:
@jtomasek updated